src/src/AdminBundle/Entity/Attendee.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace AdminBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * Attendee.
+ *
+ * @ORM\Table(name="attendee")
+ * @ORM\Entity(repositoryClass="AdminBundle\Repository\AttendeeRepository")
+ * @UniqueEntity(
+ *     fields={"deleted", "email"},
+ *     errorPath="email",
+ *     ignoreNull=false,
+ * )
+ */
+class Attendee extends BaseEntity
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="first_name", type="string", length=100)
+     * @Assert\NotBlank()
+     */
+    private $firstName;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="last_name", type="string", length=100)
+     * @Assert\NotBlank()
+     */
+    private $lastName;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="unsubscribed", type="boolean")
+     * @Assert\NotBlank()
+     */
+    private $unsubscribed = 0;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="email", type="string", length=100)
+     * @Assert\NotBlank()
+     * @Assert\Email()
+     */
+    private $email;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Language", inversedBy="attendees")
+     * @ORM\JoinColumn(name="language_id", referencedColumnName="id")
+     */
+    private $languageId;
```

Comment:
toto by sa malo volat `$language`.

Reply:
zatial som to chcela nechat tak ako to bolo aj inde, ale ok opravim. Len otazka, menila @nitrajka , alebo niekto iny aj v inych entitach tieto nazvy? ako napr. EventLanguages?

Reply:
Mergni si mastra do tvojej branche (najprv však vyrieš veci co som písal o commitoch) a uvidíš